db/migrate/20160505135144_add_volunteer_op_location_to_volunteer_op.rb

```diff
@@ -0,0 +1,5 @@
+class AddVolunteerOpLocationToVolunteerOp < ActiveRecord::Migration
+  def change
+    add_column :volunteer_ops, :volunteer_op_loc, :string
```

Comment:
given that the name of the model is VolunteerOp, what do you think about maybe calling this `address` rather than `volunteer__op_loc`?  In fact if we are intending for people to enter a street address, should we perhaps have both `address` and `postcode` on this model?